app/v2/models/user_model.py

```diff
@@ -11,7 +11,7 @@
 class User(BaseModel):
     """ model for political party """
 
-    def __init__(self, first_name=None, last_name=None, other_name=None, email=None, phone_number=None, passport_url=None, is_admin=False, password=None):
+    def __init__(self, first_name=None, last_name=None, other_name=None, email=None, phone_number=None, passport_url=None, is_admin=False, password=None, id=None):
```

Comment:
this function has 9 arguments. Consider refactoring and breaking the lines that exceed the 79th column.

Reply:
resolved